modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -69,13 +75,14 @@ kind: SelfNodeRemediationTemplate
 metadata:
 creationTimestamp: "2022-03-02T08:02:40Z"
 generation: 1
-name: self-node-remediation-default-template
+name: self-node-remediation-<remediation_object>-deletion-template <1>
 namespace: openshift-operators
 resourceVersion: "596469"
 uid: 5d29e437-c485-48fa-ba9e-0354649afd31
 spec:
 template:
 spec:
-remediationStrategy: NodeDeletion <1>
+remediationStrategy: <remediation_strategy>  <2>
 ----
-<1> Specifies the remediation strategy. The default remediation strategy is `NodeDeletion`.
+<1> Specifies the type of remediation template based on the remediation strategy, where the `<remediation_object>` either a `resource` or a `node`. For example, `self-node-remediation-resource-deletion-template`.
```

Comment:
You could split the first sentence into two lines and explicitly state what is required in place of the `<remediation_object>` attribute. For example:

> <1> Specifies the type of remediation template based on the remediation strategy. Replace `<remediation_object>` with either `resource` or `node`, for example `self-node-remediation-resource-deletion-template`.

Reply:
Done